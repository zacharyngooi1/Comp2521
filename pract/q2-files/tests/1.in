50 70 30 20 44
50 30 20 44 70
50 70 20 44 30
