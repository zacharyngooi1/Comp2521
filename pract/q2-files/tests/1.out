
Displaying t1 (sideways) 
	70
50
		44
	30
		20


Displaying t2 (sideways) 
	70
50
		44
	30
		20


Displaying t3 (sideways) 
	70
50
		44
			30
	20

 -------  
equalBST(t1, t2) returns: 1 
equalBST(t2, t3) returns: 0 
